Description: this patch disables intersphinx. 
 Intersphinx downloads stuff from internet, that is not allowed on buildds.
 .
 python-openstackclient (0.3.0-1) unstable; urgency=low
 .
   * Initial release. (Closes: #733557)
Author: gustavo panizzo <gfa@zumbi.com.ar>
Bug-Debian: http://bugs.debian.org/733557

---
Origin: vendor
Forwarded: not-needed
Reviewed-By: gustavo panizzo <gfa@zumbi.com.ar>
Last-Update: <2014-01-06>

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -31,7 +31,6 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc',
               'sphinx.ext.doctest',
-              'sphinx.ext.intersphinx',
               'sphinx.ext.todo',
               'oslosphinx']
 
